### ISSUE TITLE
Can you release the Mask Proposal Network (trained student in progressive distillation) for both base-novel and cross-dataset settings?

### ISSUE THREAD

**Issue by yxchng** [mlpc-ucsd/MasQCLIP#2]

**Comment by xuxalan** [comment]
Thanks for your interest. Since the mask proposal network is frozen in the second stage (MasQ-Tuning), you can obtain the parameters by filtering `state_dict` in our released models.
